core/src/main/scala/chisel3/internal/BiConnect.scala

```diff
@@ -10,6 +10,7 @@ import chisel3.internal.firrtl.{Connect, Converter, DefInvalid}
 
 import scala.language.experimental.macros
 import _root_.firrtl.passes.CheckTypes
+import chisel3.properties.Property
```

Comment:
Mega-nit: group with other `chisel3` imports.

Reply:
That's what I get for using VScode 😅